src/api.jl

```diff
@@ -161,13 +182,14 @@ function infer(fieldids::Vector{Tuple{Int, Int, Int}},
 
     # If there are no objects of interest, return early.
     if length(target_sources) == 0
-        return Dict{Int, Dict}()
+        return Dict{Int, Dict}(), (0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0)
```

Comment:
I don't think it needs to be visible in ```bin/celeste.jl```, does it?  In either case, simply defining it in ```src/api.jl``` should do since (for now at least) it's simply included in the Celeste module.